examples/Text Editor v1.0/TextEditor_ReadMe.txt

```diff
@@ -0,0 +1,12 @@
+My First Project in PyQt5. 
+A simple Text Editor which have these functionalities:

```

Comment:
... which *has* (not "have") these functionalities

Reply:
Yes, these errors are because of the hurry!